### PR TITLE
Remove hull-js from processor boot sequence

### PR DESCRIPTION
**Pull request by sbellity**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -9,22 +9,8 @@ import Engine from "./engine";
...
-    window.addEventListener("message", ({ data: message }) => {
-      if (message && message.event === "ship.update" && message.ship) {
-        engine.updateShip(message.ship);
-      }
-    });
-    ReactDOM.render(<App engine={engine} />, root);
-  });
+  const root = document.getElementById("app");
+  const userSearch = window.localStorage.getItem(`userSearch-${ship}`);
+  const engine = new Engine({ ship, organization, secret }, { ship: {}, userSearch });
```
Comment (timliuhull) [inline src/index.jsx]: previously we were passing ship: app which was passed from hull.js

Are we sure passing an empty object is ok?  Don't we need the code from the ship to put it in the window?  Is it getting there another way now?

Comment (timliuhull) [reply src/index.jsx]: Stephane: it’s rehydrated by the compute endpoint, here:

https://github.com/hull-ships/hull-processor/blob/develop/server/actions/compute-handler.js#L17

Comment (sbellity) [reply src/index.jsx]: it's already gracefull handled here : https://github.com/hull-ships/hull-processor/blob/develop/server/actions/compute-handler.js#L17

Comment (timliuhull) [reply src/index.jsx]: Tested the branch, for the code pull in the editor, seems to work

**Pull request merged**